<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UndupifyClass</class>
 <widget class="QMainWindow" name="UndupifyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Undupify</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>93</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectedFileLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectLocationButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>213</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Select Location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectedLocationLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Results::</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Undupify.qrc"/>
 </resources>
 <connections/>
</ui>
